Classes/GarbageCollector.php

```diff
@@ -155,6 +155,20 @@ public function processDatamap_afterDatabaseOperations($status, $table, $uid, ar
 			|| ($table == 'pages' && !$this->isIndexablePageType($record))
 		) {
 			$this->collectGarbage($table, $uid);
+
+			// if the start time is in the future and not otherwise invisible,
+			// it is necessary to add the item to the queue again to be indexed once the start time is reached
+			if ($this->isStartTimeInFuture($table, $record)
+				&& !$this->isHidden($table, $record)
+				&& !$this->hasFrontendGroupsRemoved($table, $record)
+				&& ($table != 'tt_content' && $table != 'pages_language_overlay')
```

Comment:
why do you check for these tables? Also in its current form it does not need the parentheses around the two tables.